pipelines/LSSTComCamSim/ApPipe.yaml

```diff
@@ -0,0 +1,17 @@
+description: |
+  AP pipeline specialized for Simulated ComCam Prompt Processing. This
+  file is imported in prompt_prototype/pipelines/LsstComCamSim/ApPipe.yaml
+  which is used in realtime Prompt Processing production.
+instrument: lsst.obs.lsst.LsstComCamSim
+imports:
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipe.yaml
+    exclude:
+      # TODO: prompt_prototype does not yet support R/B analysis.
```

Comment:
This TODO has now been ticketed: [DM-42220](https://jira.lsstcorp.org/browse/DM-42220).